src/maintainer/conda_forge_yml.rst

```diff
@@ -79,6 +80,34 @@ mapping for Azure-specific configuration options. For example:
       # built packages) as an Azure pipeline artifact that can be downloaded
       store_build_artifacts: False
 
+.. _self-hosted_azure-config:
+
+self-hosted azure
+-----------------
```

Comment:
This does not seem to be the right style used in this file. The sections here correspond to the acceptable top-level keys in `conda-forge.yaml`, but all of the agent settings are under the same key `azure`. I would suggest just to start with a new paragraph, followed by the config sample.
```suggestion
```